index.md

```diff
@@ -21,11 +21,11 @@ RoboJackets, Inc. is a Georgia non-profit 501(c)(7) corporation dedicated to the
 ## Officers and Directors
 The board of directors is a group of volunteer Georgia Tech alumni that provide guidance, oversight, and administrative support to the corporation. Officer positions are noted in parentheses.
 
-- Ryan Strat <small>(President)</small>
-- Jonathan Spalten <small>(Secretary)</small>
+- Jonathan Spalten <small>(President)</small>
+- Cameron Loyd <small>(Secretary)</small>
 - Kristaps Berzinch <small>(Treasurer)</small>
```

Comment:
I am no longer a directory, need to figure out how to restructure this

Reply:
oh right